packages/orbit-design-tokens/src/utils/tokensToCssVars.ts

```diff
@@ -0,0 +1,23 @@
+export const kebabCase = (str: string) => str.replace(/([a-z])([A-Z])/g, "$1-$2").toLowerCase();
```

Comment:
let's pls _not_ do these complex mappers. we **just need the palette**. let's keep it simple and maintain an object of `Record<PaletteObject, CssVar>` and keep the function _really_ simple

<img width="735" alt="Screenshot 2023-10-13 at 17 25 14" src="https://github.com/kiwicom/orbit/assets/11172408/25ec18dd-64ef-46b0-874e-9c91698c987a">

this is what I mean. this is what modules have, and this is the _only_ thing that is themable currently. let's keep an object that maps these values to CSS variables

if the palette will be extended, to e.g. `productInk` colours, we can simply add them to this object

same as in `fromPlainObject`:

```ts
interface ThemePaletteColors {
    productLight: string;
    productLightHover: string;
    productLightActive: string;
    productNormal: string;
    productNormalHover: string;
    productNormalActive: string;
    productDark: string;
    productDarkHover?: string;
    productDarkActive?: string;
    productDarker?: string;
}
interface FromPlainObject {
    (themePaletteColors: ThemePaletteColors): Tokens$1;
}
```

Reply:
fromPlainObject is not meant to be used like that. It will return you all the tokens, not just the flat object of the foundation. So if you would like to do something like: 

```
tokensToCssVars(fromPlainObject(foundation));
```

you will get the large object with all tokens transformed into css variables.

Besides that, as I already mentioned, if we would like to have this utility function to be the part of tokens package, then it should support nested objects, to make it fully compatible with our tokens and foundation. If you need a simple wrapper that will produce the css var from a plain object only, then it can be implemented right where the WL foundation is defined. 

_About complexity:_ 
  The function logic is 5 lines of code. It uses reduce and recursion, a helper function to transform from camelCase to kebabCase, sorry, but it looks quite simple and not complex at all. It's not even a cost to have a good function that fits our API and it won't be dependent on the side logic of normalization of objects from nested to flat.